071-GitHubCopilot/Student/Challenge-05.md

```diff
@@ -0,0 +1,28 @@
+# Challenge 05 - Debugging and Code Optimization with Copilot
+
+[< Previous Challenge](./Challenge-04.md) - **[Home](../README.md)** 
+
+## Introduction
+In this final challenge, you'll harness the power of GitHub Copilot to identify and resolve security vulnerabilities in a pre-written code snippet*. You will also optimize your code, refining its performance and maintainability based on the intelligent suggestions provided by Copilot. This challenge is an opportunity to deepen your understanding of how Copilot can be an integral part of your debugging and code optimization processes.
```

Comment:
why the extra "*"?

Reply:
My original intent was to make sure students were aware that this was not a security tool replacement.  However looking at it again, the paragraph is pretty short so I have removed it.